docs/api.asciidoc

```diff
@@ -152,6 +152,38 @@ func initialPageHandler(w http.ResponseWriter, req *http.Request) {
 
 See the {apm-rum-ref}/index.html[JavaScript RUM agent documentation] for more information.
 
+[float]
+[[transaction-parentid]]
+==== `func (*Transaction) ParentID() SpanID`
+
+ParentID returns the transaction's Parent ID generating a parent span ID if one has
+not already been set and tx has not been ended.
+If tx is nil or has been ended, a zero (invalid) SpanID is returned.
+
+Internally calls `Transaction.EnsureParent()`.
+
+[source,go]
+----
+// TraceContext returns a map[string]interface{} containing the trace
+// context of the transaction and span contained in ctx, if any.
+func TraceContext(ctx context.Context) map[string]interface{} {
+	tx := apm.TransactionFromContext(ctx)
+	if tx == nil {
+		return nil
+	}
+	traceContext := tx.TraceContext()
+	fields := map[string]interface{}{
+		"trace_id":              traceContext.Trace,
+		"transaction_id":        traceContext.Span,
+		"transaction_parent_id": tx.ParentID(),
+	}
+	if span := apm.SpanFromContext(ctx); span != nil {
+		fields["span_id"] = span.ParentID()
+	}
+	return fields
+}
+----
```

Comment:
I feel like all the boilerplate around the use of `span.ParentID` is a bit obscuring. Do we need an example for this? I'm not sure it's going to be commonly used (it's only come up once AFAICR), so might not be worthwhile.

Reply:
yeah, It might be too much to have an example here, I think it's fair to remove the example and see if it needs an example after it's been released.